src/qt/test/apptests.cpp

```diff
@@ -22,6 +22,7 @@
 #include <QAction>
 #include <QEventLoop>
 #include <QLineEdit>
+#include <QRegularExpression>
```

Comment:
https://github.com/bitcoin/bitcoin/pull/17213#issuecomment-546684680

Reply:
I verified that Qt itself uses `QRegularExpression` header, at least for version we use in depends.

So, I'm ok with it.